docs/source/Schema.md

```diff
@@ -123,15 +123,31 @@ Projects can provide sets of exemplar answers for use in test scripts. e.g.:
 Documents
 ---------
 
-Documents occur as `introduction` and `output` documents of Modules as well as Question prompts. A document appears as:
+Documents occur as `introduction` and `output` documents of Modules, and a restricted for of documents also occurs in Question prompts. A document appearing in the output documents list is given as:
```

Comment:
Typo in sentence? "and a restricted for of documents" doesn't make sense.

Reply:
Should be "form." Fixing...